apps/core-app/src/pages/Proposals.tsx

```diff
@@ -13,13 +24,24 @@ const LeftCard = styled(Card)`
   }
 `;
 
+export const VALID_NEW_PROPOSALS = [FORM.SIGNAL, FORM.SHARE_SWAP];
```

Comment:
Can we just do object lookup in our FORM object?

Reply:
> Can we just do object lookup in our FORM object?

I wonder if we want more fine grain control over which forms are on this list?